lib/resources/elasticsearch.rb

```diff
@@ -0,0 +1,140 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier, rx294@nyu.edu
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
+
+require 'utils/filter'
+
+module Inspec::Resources
+  class Elasticsearch < Inspec.resource(1)
+    name 'elasticsearch'
+    desc 'See Description below'
+    # Use the Elasticsearch InSpec audit resource to test the Elasticsearch
+    # configuration settings. The data is extract from the live elastic search
+    # cluster using the curl command `curl localhost:9200/_nodes/`. The resource
+    # makes uses of FilterTable and its functionality can be used
+
+    example 'See example below'
+    # elasticsearch.where{ cluster_name == 'elasticsearch' }.version.each do |node_version|
+    #   describe node_version do
+    #     it {should eq '5.4.1'}
+    #   end
+    # end
+
+    attr_reader :params, :content
+
+    def initialize(es_ip = 'localhost', es_port = 9200)
+      @es_ip   = es_ip
+      @es_port = es_port
+
```

Comment:
You are not guaranteed for `curl` to be on the target system, so you should check for that here, and `skip_resource` if it's not available.